blocks.go

```diff
@@ -21,16 +21,12 @@ package emulator
 import (
 	"errors"
 
-	"github.com/onflow/flow-go/access"
 	"github.com/onflow/flow-go/fvm"
 	flowgo "github.com/onflow/flow-go/model/flow"
 
 	"github.com/onflow/flow-emulator/storage"
 )
 
-var _ fvm.Blocks = &blocks{}
-var _ access.Blocks = &blocks{}
```

Comment:
Why remove the type assertions?

Reply:
Ohh, I thought this was meant to silent an unused warning :/ sorry not yet familiar with some patterns in Go. Will revert.